h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -1,16 +1,29 @@
 import GroupFormHeader from './GroupFormHeader';
 import type { Group } from '../config';
 import FormContainer from './forms/FormContainer';
+import { useState } from 'preact/hooks';
```

Comment:
This should be in its own group of imports at the top.

Reply:
Actually, I just noticed the imports are not properly sorted here 🤔 

I think we may be missing the import sorting prettier plugin in this project

Reply:
I created this PR to make sure we keep imports sorted automatically https://github.com/hypothesis/h/pull/9550